apollo-client/src/main/java/com/ctrip/framework/apollo/kubernetes/KubernetesManager.java

```diff
@@ -205,4 +223,43 @@ public boolean checkConfigMapExist(String k8sNamespace, String configMapName) {
             return false;
         }
     }
+
+    /**
+     * check pod whether pod can write configmap
+     *
+     * @param k8sNamespace config map namespace
+     * @return true if this pod can write configmap, false otherwise
+     */
+    private boolean isWritePod(String k8sNamespace) {
+        try {
+            if (Strings.isNullOrEmpty(localPodName)) {
+                return true;
+            }
+            V1Pod localPod = coreV1Api.readNamespacedPod(localPodName, k8sNamespace, null);
+            V1ObjectMeta localMetadata = localPod.getMetadata();
+            if (localMetadata == null || localMetadata.getLabels() == null) {
+                return true;
+            }
+            String appName = localMetadata.getLabels().get("app");
+            String labelSelector = "app=" + appName;
+
+            V1PodList v1PodList = coreV1Api.listNamespacedPod(k8sNamespace, null, null,
```

Comment:
When the number of pod instances is large, the interface return value may be particularly large, and the limit parameter needs to be used

Reply:
I don't know what is a large number of instances, so I  set it to 1000 . How about your advice?